services/post-service.js

```diff
@@ -12,8 +13,15 @@ class PostService {
     return post;
   }
 
-// ------------------------------ Get Posts ------------------------------ //
-
+  // TODO: example of documentation
+  /**
+   * Applies paginated filters to the posts.
+   *
+   * @param userId
+   *   User id.
+   * @param postedBy
+   *   Author id.
+   */
```

Comment:
_hint:_ there also must be pagination object that include skip, limit and page or **you can use cursor pagination**